### PR TITLE
Clarified DCO and CLA requirements

### PR DESCRIPTION
**Pull request by bensternthal**
Added clarifications for DCO & CLA, _hoping_ to address #387, #632, and #778

### CODE REVIEW

File: governance/IP_POLICY_GUIDANCE.md [hunk 1]
```diff
@@ -53,15 +59,27 @@ All project websites need to have a [standard website footer][].
...
+#### DCO
+
+There are two options for implementing a DCO on your project:
+
+1. Via installing the [DCO bot](https://github.com/apps/dco) 
+2. By including the following in your repository:
+   1. Adding the DCO to your CONTRIBUTING.md file, [example](https://github.com/nodejs/node/blob/main/CONTRIBUTING.md)
+   2. Adding the DCO to your PR templates, [example](https://raw.githubusercontent.com/nodejs/node/main/.github/PULL_REQUEST_TEMPLATE.md)
+
+The DCO Bot is the preferred method and is mandatory for any new projects joining the foundation. We also recommend this method for existing projects that require stricter enforcement.
```
Comment (tobie) [inline governance/IP_POLICY_GUIDANCE.md:71]: Are there any downsides to using the bot?

If there aren't, I think that's the only option we should be listing.

**Review by tobie — COMMENTED** [review]
Thanks for making this happen! Unbelievable!!

Would love to understand if there are downsides to using the bot for the DCO.

File: governance/IP_POLICY_GUIDANCE.md (hunk 1 above)
Comment (timmywil) [reply governance/IP_POLICY_GUIDANCE.md:71]: Not everyone trusts bots with permissions. I think it's good to provide an alternative.

Comment (timmywil) [reply governance/IP_POLICY_GUIDANCE.md:71]: > The DCO Bot is the preferred method and is mandatory for any new projects joining the foundation

I wouldn't make this mandatory. Besides, this statement conflicts with the one above about *either* adopting a DCO or CLA. If going the CLA route, the DCO bot would not be mandatory.

Comment (tobie) [reply governance/IP_POLICY_GUIDANCE.md:71]: Permissions requested are fairly lightweight:

- Read access to code, metadata, and pull requests
- Read and write access to checks

I'm genuinely curious what the concern is. (Not trying to argue for the bot, just trying to understand.)

Comment (ljharb) [reply governance/IP_POLICY_GUIDANCE.md:71]: I’m not concerned about permissions; i don’t want a non-human entity speaking prose to my contributors.

Comment (mhdawson) [reply governance/IP_POLICY_GUIDANCE.md:71]: Unless something has changed there were concerns with using the DCO bot when it was proposed for the Node.js project a number of years ago. Brian (formerly OpenJS staff) was going to try to PR in changes to address those concerns but I don't think they every landed.

Comment (mcollina) [reply governance/IP_POLICY_GUIDANCE.md:71]: The problem with the bot is that most people don't know how to DCO commits, rebase, or do any advanced git tasks. Essentially, it forcefully raises the barrier to contributions. I have the bot enable on Platformatic, and every other contributors can't fix it.

**Comment by tobie** [comment]
**Notes from today's CPC call:**

We assumed that @bensternthal would come back with an update to the PR accounting for the comments received. 

_@bensternthal, please do let us know if you had something different in mind._

**Comment by bensternthal** [comment]
@tobie Thanks for keeping me on my toes. Adjusted the PR with feedback.

File: governance/IP_POLICY_GUIDANCE.md [hunk 2]
```diff
@@ -53,15 +53,27 @@ All project websites need to have a [standard website footer][].
...
+#### CLA
+
 The Board has pre-approved an [individual CLA][ICLA] and a [corporate CLA][CCLA].
 Projects can choose to adopt either the individual CLA by itself or both the individual CLA and the corporate CLA without any further review.
 
 If you need to use a different CLA, you must
 [obtain an exemption from the board](#obtaining-an-exemption-from-the-board).
 
-We are preparing additional guidance documentation on how to implement the DCO or a CLA, what infrastructure the foundation provides to help with this, what steps maintainers need to make when merging pull requests, and how to fix common issues that arise when using either solution.
+CLAs are managed through [EasyCLA](https://docs.linuxfoundation.org/lfx/easycla). Please reach out to the [OpenJS staff](mailto:operations@openjsf.org) for assistance with implementation.
```
Comment (tobie) [inline governance/IP_POLICY_GUIDANCE.md:75]: Let's move all of the relevant contact details in the "Getting help" section and maybe split it into two emails there (or any other structure that makes sense to you from an operational perspective), eg:

```md
If you have any questions about how to _implement_ this policy,  please reach out to [operations@openjsf.org](mailto:operations@openjsf.org).
If you have any _other questions_ about this policy, please reach out to [legal-questions@lists.openjsf.org](mailto:legal-questions@lists.openjsf.org).
```

WDYT?

**Review by mertcanaltin — APPROVED** [review]
lgtm

**Pull request merged**